hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/SparkFileFormatInternalRowReaderContext.scala

```
@@ -249,7 +257,7 @@ class SparkFileFormatInternalRowReaderContext(parquetFileReader: SparkParquetRea
               sparkAdapter.makeColumnarBatch(vecs, s.numRows())
```

Should a `TODO` be left here to support columnar batch merging with partition field projection in the future?

---

Yes, do we want to throw an error if we have some partition values but hit this case?

---

Yes, let's do that so user knows the limitation.